system/jlib/jptree.cpp

```diff
@@ -8804,37 +8855,31 @@ MODULE_EXIT()
     ::Release(configFileUpdater);
 }
 
-unsigned installConfigUpdateHook(ConfigUpdateFunc notifyFunc)
+unsigned installConfigUpdateHook(ConfigUpdateFunc notifyFunc, bool callWhenInstalled)
 {
-    if (!configFileUpdater)
-        WARNLOG("installConfigUpdateHook(): configuration updater not installed");
-    else
-        return configFileUpdater->addNotifyFunc(notifyFunc);
-    return 0;
+    if (!configFileUpdater) // NB: installConfigUpdateHook should always be called after configFlieUpdater is initialized
```

Comment:
picky typo: configFileUpdater (and elsewhere)

Reply:
Will fix in new PR: https://github.com/hpcc-systems/HPCC-Platform/pull/17012